pyfar/dsp/dsp.py

```diff
@@ -223,3 +224,75 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def deconv(measurement, excitation, **kwargs):
+    """Function to calculate the deconvolution of two signals.
```

Comment:
This comment is a bit picky - I think the one line docstring guideline is to start with a verb :)
```suggestion
    """Calculate transfer functions by spectral deconvolution of two signals.
```

Reply:
check. :)

Reply:
check. :)